pootle/apps/accounts/management/commands/update_user_email.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import CommandError
+from django.core.validators import validate_email, ValidationError
+
+from allauth.account.models import EmailAddress
+
+from . import UserCommand
+
+
+User = get_user_model()
+
+
+class Command(UserCommand):
+    args = "user email"
+    help = "Update user email address"
+
+    def handle(self, *args, **kwargs):
+        super(Command, self).handle(*args, **kwargs)
+        user = self.get_user(args[0])
+        new_email = args[1]
```

Comment:
Just noticed it is done below. I was expecting it to be validated before used, so IMHO my confusion is understandable.